db.js

```diff
@@ -515,15 +509,17 @@ var users = (function () {
       });
     },
     insert: function (user) {
-      var sql = "INSERT INTO users (name, google_id, picture) VALUES (?,?,?)";
-      var inserts = [user.name, user.google_id, user.picture];
+      var sql =
+        "INSERT INTO users (name, google_id, picture, email) VALUES (?,?,?,?)";
+      var inserts = [user.name, user.google_id, user.picture, user.email];
       sql = mysql.format(sql, inserts);
 
       return pool.query(sql);
     },
     update: function (user) {
-      var sql = "UPDATE users SET name = ?, picture = ? WHERE id = ?";
-      var inserts = [user.name, user.picture, user.id];
+      var sql =
+        "UPDATE users SET name = ?, picture = ?, email = ? WHERE id = ?";
+      var inserts = [user.name, user.picture, user.id, user.email];
```

Comment:
I would also search for everywhere this is used to make sure it's passing in the right things. You can use VSCode's Right Click -> Find All References to do this pretty easily.